Dockerfile.template

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2
+
+RUN set -ex \
+	&& mkdir -p /opt/changelog-generator \
+	&& apt-get update -yq \
+	&& buildDeps=' \
+		libffi-dev \
+		ruby-dev \
+		build-essential \
+	' \
+	&& apt-get install -y ruby \
+	&& apt-get install -y ${buildDeps} \
+	&& git clone -b %%GCG_VERSION%%-azavea https://github.com/azavea/github-changelog-generator \
```

Comment:
A bit of a nitpick, but I think it would be better to download the tarball of this branch vs. `git clone`:

```
https://github.com/azavea/github-changelog-generator/archive/v1.14.3-azavea.tar.gz
```